src/components/consul/detail/ConsulCommentListItem.js

```diff
@@ -36,14 +36,14 @@ const deleteCommentAlert = (onDelete) =>
 
 /* eslint-disable complexity */
 /* NOTE: we need to check a lot for presence, so this is that complex */
-export const ConsulCommentListItem = ({ commentItem, onRefresh, replyList, navigation }) => {
-  const [showResponse, setShowResponse] = useState(false);
-  const [showReply, setShowReply] = useState(false);
+export const ConsulCommentListItem = ({ commentItem, refetch, replyList, navigation }) => {
   const [isLoading, setIsLoading] = useState(false);
+  const [isShowReply, setIsShowReply] = useState(false);
+  const [isShowResponse, setIsShowResponse] = useState(false);
```

Comment:
`show` would be enough to indicate a boolean. it does not have to be `is` all the time. see it as natural language when you ask for something.

1. is something loading? -> `isLoading`?
2. should replies be shown? -> `showReply`?
3. should responses be shown? -> `showResponses`?

questions for conditions could be understand very fine i guess. with `is` and `show` at the same time, it seems duplicated for me and i have difficulties to make a natural sentence. you know what i mean?

Reply:
Yes, I understand better. Thank you.